android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -75,21 +117,25 @@ public AssetLoader(@NonNull Engine engine, @NonNull MaterialProvider generator,
         }
     }
 
+    /** Frees all memory consumed by the native AssetLoader and its material cache. */
     public void destroy() {
         nDestroyAssetLoader(mNativeObject);
         mNativeObject = 0;
     }
 
+    /** Creates a FilamentAsset from the contents of a GLB file. */
```

Comment:
Link to other classes with `{@link FilamentAsset}` or at least use `<code>FilamentAsset</code>` to distinguish APIs from regular text.